rosen_backend/siteConfig/settings.py

```diff
@@ -76,20 +76,20 @@
 # Database
 # https://docs.djangoproject.com/en/4.1/ref/settings/#databases
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
-    }
-}
-
 # DATABASES = {
 #     'default': {
-#         'ENGINE': 'djongo',
-#         'NAME': 'rosenDB',
+#         'ENGINE': 'django.db.backends.sqlite3',
+#         'NAME': BASE_DIR / 'db.sqlite3',
```

Comment:
Shouldn't this be the part that is uncommented for now?

Reply:
Yeah do not merge this, this has to be uncommented

Reply:
Okay, I'll closes this PR, so nobody accidentally close